build_windows.md

```diff
@@ -1,65 +1,97 @@
+
 # Building the Podman client and client installer on Windows
 
-The following describes the process for building the Podman client on Windows.
+The following describes the process for building and testing the Podman Windows client (`podman.exe`) and the Podman Windows installer (`podman-setup.exe`) on Windows.
+
+## Topics
+- [Requirements](#requirements)
+    - [OS requirements](#os-requirements)
+    - [Git and go](#git-and-go)
+    - [Pandoc](#pandoc)
+    - [WiX Toolset v3](#wix-toolset-v3)
+    - [Virtualization Provider](#virtualization-provider)
+        - [WSL](#wsl)
+        - [Hyper-V](#hyper-v)
+- [Get the source code](#get-the-source-code)
+    - [Allow local PowerShell scripts execution](#allow-local-powershell-scripts-execution)
+- [Build and test the Podman client for Windows](#build-and-test-the-podman-client-for-windows)
+    - [Build the Podman client](#build-the-podman-client)
+    - [Download gvproxy.exe and win-sshproxy.exe](#download-gvproxyexe-and-win-sshproxyexe)
+    - [Create a configuration file (optional)](#create-a-configuration-file-optional)
+    - [Create and start a podman machine](#create-and-start-a-podman-machine)
+    - [Run a container using podman](#run-a-container-using-podman)
+- [Build and test the Podman Windows installer](#build-and-test-the-podman-windows-installer)
+    - [Build the Windows installer bundle](#build-the-windows-installer-bundle)
+- [Validate changes before submitting a PR](#validate-changes-before-submitting-a-pr)
+
+## Requirements
+
+### OS requirements
+
+Windows OS can behave very differently depending on how it was configured. This documentation assumes that one is using a [Windows 11 development machine](https://developer.microsoft.com/en-us/windows/downloads/virtual-machines/) or a configuration close to this one. The Podman Windows client installer bundles several tools, which are unnecessary for Podman builds, but this
+set of packages is well aligned with GitHub's `windows-latest` offerings. Some of the tools will still be missing from this distribution and will have to be manually added after this installation completes.
+
+### Git and go
+
+To build Podman, the [git](https://gitforwindows.org/) and [go](https://go.dev) tools are required. In case they are not yet installed,
+open a Windows Terminal and run the following command (it assumes that [winget](https://learn.microsoft.com/en-us/windows/package-manager/winget/) is installed):
```

Comment:
```suggestion
open a Windows PowerShell terminal and run the following command (it assumes that [winget](https://learn.microsoft.com/en-us/windows/package-manager/winget/) is installed):
```

I think we should recommend PowerShell over cmd-prompt, since that's usually what we've been testing with?